### ISSUE TITLE
[Question] Is there any plan to support file concat

### ISSUE THREAD

**Issue by ian4hu** [cdnjs/cdnjs#9537]
If I use the `nginx_concat_module` I can concat serveral js/css file in a single request like below
```html
        <!-- Polyfills -->
        <!--[if lt IE 10]>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/??console-polyfill/0.2.2/index.js,es5-shim/4.5.7/es5-shim.min.js,es5-shim/4.5.7/es5-sham.min.js,es6-shim/0.35.1/es6-sham.min.js,es6-shim/0.35.1/es6-shim.min.js,html5shiv/3.7.2/html5shiv.min.js,media-match/2.0.2/media.match.min.js"></script>
        <![endif]-->
```

and that reduce the times of request.

But I've found that cdnjs is not support that now, Is there any plan to support it?

**Comment by PeterDaveHello** [comment]
@ian4hu by supporting http/2.0 and spdy, there is no need to concat the files, to concat files can help improve performance since http/1 doesn't support multiplex/parallel the connections, and have the limitation on the connection numbers, however it'll may you not benefit from cache sharing by concating the files, In http/2.0, the connection would be reused, which means the less tcp 3-way hand-shaking overhead, and the content will be downloaded parallelly, the files will have their own cache and can leverage from cache sharing.

In summary, http/2.0 without concating will be faster then concating in both http/2.0 or http/1!

References:
 - https://en.wikipedia.org/wiki/HTTP/2
 - https://http2.github.io/

**Comment by ian4hu** [comment]
@PeterDaveHello  Thanks for your response, I got it, and the http/2 is better than concat, closed.

**Comment by PeterDaveHello** [comment]
@ian4hu Thanks.
